packages/desktop-client/src/components/reports/spreadsheets/age-of-money-spreadsheet.ts

```
@@ -0,0 +1,360 @@
+ */
+function getPeriodKey(
+  date: string,
+  granularity: AgeOfMoneyGranularity,
+): string {
+  const parsed = d.parseISO(date);
+  switch (granularity) {
+    case 'daily':
+      return date; // YYYY-MM-DD
+    case 'weekly':
+      // Use start of week (Monday) as key
+      const weekStart = d.startOfWeek(parsed, { weekStartsOn: 1 });
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)